web/packages/teleport/src/Main/Main.tsx

```diff
@@ -299,6 +321,7 @@ export const HorizontalSplit = styled.div`
   display: flex;
   flex-direction: column;
   flex: 1;
+  ${props => props.dockedView && 'max-width: calc(100% - 520px);'}
```

Comment:
I'm seeing this `520px` hardcoded a few times in the code, could we export it to `sharedStyles` and reference it from there? (in `Assist.tsx` as well 🙏 )